api/main_endpoints/routes/User.js

```diff
@@ -525,4 +526,39 @@ router.post('/usersValidVerifiedAndSubscribed', function(req, res) {
     });
 });
 
+// Generate an API key for the Messages API if the user does not have an API key; otherwise, return the existing API key
+router.post(('/retrieveAPIKey'), async (req, res) => {
```

Comment:
i wanna use this endpoint from the command line, lets make it simple
```suggestion
router.post(('/apikey'), async (req, res) => {
```